src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -649,6 +651,16 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("INSERT VERSION");
 		}
+		Skript.registerEvent("Potion Brewing", SimpleEvent.class, BrewEvent.class, "[potion] brew[ing]")
```

Comment:
I suggest adding Skript's old planned syntax `[potion] brew[ing] [of %itemtypes%]` ItemType because it's a literal.